packages/theme/components/CartSidebar.vue

```diff
@@ -0,0 +1,262 @@
+<template>
+  <div id="cart">
+    <SfSidebar
+      v-e2e="'sidebar-cart'"
+      :visible="isCartSidebarOpen"
+      title="My Cart"
+      class="sf-sidebar--right"
+      @close="toggleCartSidebar"
+    >
+      <template #content-top>
+        <SfProperty
+          v-if="totalItems"
+          class="sf-property--large cart-summary desktop-only"
+          name="Total items"
+          :value="totalItems"
+        />
+      </template>
+      <transition name="sf-fade" mode="out-in">
+        <div v-if="totalItems" key="my-cart" class="my-cart">
+          <div class="collected-product-list">
+            <transition-group name="sf-fade" tag="div">
+              <SfCollectedProduct
+                v-for="product in products"
```

Comment:
Please add :key

Reply:
Key is already there -> `:key="cartGetters.getItemSku(product)"`